k8s/object.bzl

```diff
@@ -89,6 +89,11 @@ def _impl(ctx):
                 for (k, v) in image_spec.items()
             ])]
 
+    # Add workspace_status_command files to the args that are pushed to the resolver and adds the
+    # files to the runfiles so they are available to the resolver executable.
+    stamp_inputs = [ctx.info_file, ctx.version_file]
+    stamp_args = " ".join(["--stamp-info-file=%s" % _runfiles(ctx, f) for f in stamp_inputs])
```

Comment:
do 
`all_inputs += stamp_inputs ` here so you dont need to change line 136

Reply:
done :)